README.md

```diff
@@ -61,6 +61,11 @@ print("Result memory:", memory)
 print("Result counts:", counts)
 ```
 
+### Rigetti Quantum Cloud Services (QCS)
+
+Execution against a QPU requires a [reservation via QCS](https://docs.rigetti.com/qcs/guides/reserving-time-on-a-qpu).
```

Comment:
I'd consider omitting this first link since it's likely to fall stale (unless we're ok with the "not found" page of the QCS docs, which now that I look at it isn't so bad). 
![image](https://user-images.githubusercontent.com/22137047/127371744-43ca475f-955b-460d-ae70-7c4ec042593f.png)

Reply:
I'm OK with that page for now